ecs-init/docker/docker.go

```diff
@@ -427,6 +432,65 @@ func getDockerPluginDirBinds() []string {
 	return pluginBinds
 }
 
+// take pathPredicate as an argument for unit testing
+func getCapabilityExecBinds(pathPredicate func(path string, predicate func(fileInfo os.FileInfo) bool) (bool, error)) []string {
+	const (
```

Comment:
if these constants are moved out of this method, they can be used in unit tests as well :)

Reply:
good point, was trying to create some encapsulation so it's easier to read, will move those out